apache-tika-3.0.yaml

```diff
@@ -1,13 +1,14 @@
 package:
   name: apache-tika-3.0
   version: 3.0.0
-  epoch: 2
+  epoch: 3
   description: The Apache Tika toolkit detects and extracts metadata and text from over a thousand different file types (such as PPT, XLS, and PDF).
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
-      - openjdk-17-jre
+      - dash-binsh
+      - openjdk-17-default-jvm
```

Comment:
Is whole jvm required here or we can work with jre only? 💭

Reply:
as they use "exec java" in their entrypoint in the image config https://github.com/apache/tika-docker/blob/main/full/Dockerfile#L77 its necessary to add jvm here 🫠

Reply:
that looks good 🫠

the CVE check is failing as of now, do we need to file advisories for it if it's not fixable by bumping the deps?

Reply:
so the PR is in draft mode, I'm going to mark it as ready for review once the local tests are succeed @kranurag7

Reply:
its ready for review brother, thank you!